packages/react-notion-x/package.json

```diff
@@ -30,7 +30,8 @@
     "react-lazy-images": "^1.1.0",
     "react-modal": "^3.14.3",
     "react-pdf": "^5.4.1",
-    "react-use": "^17.3.1"
+    "react-use": "^17.3.1",
+    "next": "^12.0.2"
```

Comment:
`next` shouldn't be a dependency of `react-notion-x`.

Currently, `react-notion-x` isn't tied to next.js (it can be used with any react-based web framework).

If we were to tie it to next.js, it still wouldn't be a dependency of `react-notion-x`; it would be a peer dep or an optional dep.

Reply:
for an example of how to do this, see how we're handling `next/link` via the custom `pageLink` prop so that it's not part of the package itself but rather overridable by the client. https://github.com/transitive-bullshit/nextjs-notion-starter-kit/blob/main/components/NotionPage.tsx#L228

Reply:
I made a new custom prop `image` to use custom images components, like next/image! 695690f

Reply:
I'm still working on how to perfect the medium-zoom component to work properly with next/image

Reply:
Just pushed a new commit to allow custom preview images! 3345717